City-of-Helsinki/helsinki-paatokset
# PP-288: Decision page fixes

**To test**
* Checkout branch, run `make drush-cr`
* If you don't have Ahjo data imported yet, run: `drush ap:fs;drush mim ahjo_cases:latest;drush mim ahjo_decisions:latest;drush mim ahjo_meetings:latest;drush mim ahjo_decisionmakers:all;drush ap:ud --localdata`
* Good decision to test with: https://helsinki-paatokset.docker.so/fi/asia/hel-2020-006250
* The "Download PDF" button should now have the correct color (and also hover and focus colors)
* The decision date should now be visible in the "Appeal process accordion" and the text should be translatable
* The "Presenter info" section should be moved below the accordion
* The content (also inside accordions) should have consistent font sizes (tables, div elements, etc).
* Any UL and OL elements should be inside the content area
* The dropdown should have correct color coding on all options
* The dropdown shouldn't have double hover effects anymore
* The "next" and "previous" buttons should work correctly and you should be able to navigate to the end and beginning of the list using these (previously the order was sort of random)
* Check one of the "other decisions for this meeting" links (some should have available meetings). The arrow icon should have consistent sizing and positioning now
